### PR TITLE
Add go.mod

### PR DESCRIPTION
**Pull request by greut**
Add a go module definition with this address.

Without it, the following `replace` has to be used.

```
replace github.com/saintfish/chardet => github.com/gogs/chardet v0.0.0-20150115103509-2404f7772561
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1 @@
+module github.com/saintfish/chardet
```
Comment (unknwon) [inline go.mod]: It should match the import path `github.com/gogs/chardet`.

**Review by unknwon — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (greut) [reply go.mod]: Alright, I'll change the tests then.

Comment (unknwon) [reply go.mod]: Thanks for making this PR!

**Comment by unknwon** [comment]
Thank you!

**Review by unknwon — APPROVED** [review]

**Pull request merged**